starter-code/server.js

```diff
@@ -122,6 +133,7 @@ app.delete('/articles/:id', function(request, response) {
 app.delete('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
```

Comment:
.5 - (server.js) ln. 123 “DELETE: Articles route for removing all records from the DB (does not delete the table)” You're really close here. It's not deletRecord but something simular.